tests/robot-tests/tests/admin_and_public/bau/publication_owner.robot

```diff
@@ -0,0 +1,340 @@
+    user selects from list by label  css:[name="selectedReleaseRole"]  Viewer
+    user clicks button  Add release access
+    
+Sign in as Analyst1 User1 & navigate to publication as a viewer 
+    [Tags]  HappyPath
+    user signs in as analyst1
+    user goes to url  %{ADMIN_URL}
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}  120
+
+Sign in as Analyst1 User1 & navigate to publication
+    [Tags]  HappyPath
+    user signs in as analyst1
+    user goes to url  %{ADMIN_URL}
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}  120
+    
+Navigate to release as a viewer
+    [Tags]  HappyPath
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+
+    ${accordion}=  user gets accordion section content element   ${PUBLICATION_NAME}
+    ${details}=  user gets details content element  Academic Year 2025/26 (Live - Latest release)  ${accordion}
+
+    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
+    ${edit_button}=  get child element  ${details}  xpath:.//a[text()="View this release"]
+    user clicks element   ${edit_button}
+
+    user waits until h2 is visible  Release summary
+    user checks summary list contains   Publication title  ${PUBLICATION_NAME}
+
+Navigate to administration as bau1 to assign publication owner only access
+    [Tags]  HappyPath
+    user signs in as bau1
+    user goes to url  %{ADMIN_URL}/administration/users
+
+Navigate to manage users as bau1
+    [Tags]  HappyPath
+    user waits until element is enabled  //*[tbody]//tr[1]//td[3]//a
+    user clicks element  //*[tbody]//tr[1]//td[3]//a
+    user waits until page does not contain loading spinner
+    
+Remove contributor access from Analyst1 
+    [Tags]  HappyPath
+    user clicks element  //*[@data-testid="releaseAccessViewer"]
+    user waits until page does not contain loading spinner
+    Sleep  0.2
+
+Give Analyst1 User1 publication owner role
+    [Tags]  HappyPath
+    user waits until element is enabled  css:[name="selectedPublicationId"]
+    user selects from list by label  css:[name="selectedPublicationId"]  ${PUBLICATION_NAME}
+
+    user waits until element is enabled  css:[name="selectedPublicationRole"]
+    user selects from list by label  css:[name="selectedPublicationRole"]  Owner
+    user clicks button  Add publication access
+    user waits until page does not contain loading spinner
+    Sleep  0.2    
+
+Navigate to admin url as a publication owner
+    [Tags]  HappyPath
+    user signs in as analyst1 as a publication owner 
```

Comment:
Can just use `user signs in as analyst1` here

Reply:
As I've said in a previous comment this assertion will fail on the following: 

```
  user waits until page contains element   id:publicationsReleases-themeTopic-themeId   180
    user checks breadcrumb count should be  2
    user checks nth breadcrumb contains  1   Home
    user checks nth breadcrumb contains  2   Administrator dashboard

```